idaes/models_extra/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -446,10 +452,97 @@ def initialize(
     def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
 
+        # Get units meta data from property packages (only solid needed)
+        units_meta_solid = (
+            self.config.solid_property_package.get_metadata().get_derived_units
+            )
+
+        # scale some variables
+        if hasattr(self, "bed_diameter"):
+            if iscale.get_scaling_factor(self.bed_diameter) is None:
+                sf = 1 / value(self.bed_diameter)
+                iscale.set_scaling_factor(self.bed_diameter, 1e-1 * sf)
```

Comment:
It seems a bit odd that your are multiplying the inverse of the current value by 0.1 to get the scaling factor. A comment on why htis is necessary would probably be useful.

Reply:
@bpaul4 it is okay to remove this.